iron-collapse.html

```diff
@@ -174,19 +182,23 @@
         // For `auto` we must calculate what is the final size for the animation.
         // After the transition is done, _transitionEnd will set the size back to `auto`.
         if (size === 'auto') {
```

Comment:
I'd rather update the callers of this function to pass `""` instead of `"auto"`. This should allow you to just change this check to `if (!size)`